cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ApplicationLifecycleService.java

```diff
@@ -86,15 +86,15 @@
 import io.cdap.cdap.internal.capability.CapabilityNotAvailableException;
 import io.cdap.cdap.internal.capability.CapabilityReader;
 import io.cdap.cdap.internal.profile.AdminEventPublisher;
-import io.cdap.cdap.messaging.spi.MessagingService;
 import io.cdap.cdap.messaging.context.MultiThreadMessagingContext;
+import io.cdap.cdap.messaging.spi.MessagingService;
```

Comment:
I'm surprised this could get merged, are we missing something in the checkstyle?

Reply:
Not sure. Let me check the original PR

Reply:
It was reported in https://github.com/cdapio/cdap/pull/15375/files but it was a pretty big PR so I think the checkstyle warnings were ignored